clients/drcachesim/common/options.cpp

```diff
@@ -42,6 +42,17 @@ droption_t<bool> op_offline
  "If this option is enabled, trace data is instead written to files in -outdir "
  "for later offline analysis.  No simulator is executed.");
 
+droption_t<bytesize_t> op_num_threads
+(DROPTION_SCOPE_ALL, "num_threads", 0, "Number of threads for writing traces",
+ "For the offline analysis mode (when -offline is requested), specifies the number "
+ "of sideline threads are used to write traces file out.  "
+ "0 means synchronized write without any sideline threads.");
+
+droption_t<unsigned int> op_queue_size
+(DROPTION_SCOPE_ALL, "queue_size", 4000, "The queue size for ",
+ "For the offline analysis mode (when -offline is requested), specifies the size "
+ "of the queue for batching trace data to be written out by sideline threads.");
```

Comment:
What are the units here?  If bytes this is way too small.  If buffers it seems large for a default: 128MB for 32K buffers.